// https://protobuf.dev/programming-guides/proto3
/*
python -m grpc_tools.protoc \
  --proto_path=./src/cmd/grpc/idl \
  --python_out=./src/cmd/grpc/idl \
  --pyi_out=./src/cmd/grpc/idl \
  --grpc_python_out=./src/cmd/grpc/idl \
  ./src/cmd/grpc/idl/synthesize.proto
*/

syntax = "proto3";

package chat_bot.tts;

message SynthesizeRequest {
  string tts_text = 1;
  string tts_tag = 2;
  map<string, string> kawa_params = 3;
}

message SynthesizeResponse { bytes tts_audio = 1; }

service TTS {
  rpc SynthesizeUS(SynthesizeRequest) returns (stream SynthesizeResponse) {}
}
